src/jit/decomposelongs.cpp

```diff
@@ -878,12 +878,6 @@ GenTree* DecomposeLongs::DecomposeNeg(LIR::Use& use)
     GenTree* gtLong = tree->gtGetOp1();
     noway_assert(gtLong->OperGet() == GT_LONG);
 
-    LIR::Use op1(Range(), &gtLong->gtOp.gtOp1, gtLong);
-    op1.ReplaceWithLclVar(m_compiler, m_blockWeight);
-
-    LIR::Use op2(Range(), &gtLong->gtOp.gtOp2, gtLong);
-    op2.ReplaceWithLclVar(m_compiler, m_blockWeight);
```

Comment:
If you remove these then you'll likely need to change the line below that drops the side effect flags. `loResult` should get `loOp1`'s side effects and `hiResult` should get `hiOp1`'s side effects.